README.md

```diff
@@ -53,6 +53,11 @@ which is supported by the tldr-pages project maintainers:
 
     npm install -g tldr
 
+Or you can use the just as easy to install [python3 client](https://github.com/tldr-pages/tldr-python-client) which is also supported by the tldr-pages project maintainers, however it is available on pip3 which is more likely to be on your computer:
```

Comment:
This is quite subjective. Specifically `however it is available on pip3 which is more likely to be on your computer` - that all depends on people's setups. Can we make this more objective perhaps?

Reply:
Yes it is subjective, however most linux distributions have it, and Arch has its community repo  package.

And who has NPM and not python PIP.

I imagine that PIP has a lot more downloads compared to NPM

Reply:
I'll find a solution

Reply:
Or you can use the just as easy to install [python3 client](https://github.com/tldr-pages/tldr-python-client) which is also supported by the tldr-pages project maintainers, however it is available on pip which you may have installed over npm

Reply:
How about this.
```suggestion
You can also use the [official Python client](https://github.com/tldr-pages/tldr-python-client), which is supported by the tldr-pages project maintainers as well.
```

Reply:
Yes that works completely however I feel that the average newb, especially Linux users might have not have NPM but might have pip preinstalled.

And from what I have seen, the python client from pip is better than the haskell client in multiple ways for example customization and command passing, eg `tldr git what-changed` doesn't work

Reply:
How about
```suggestion
You can also use the [official Python client](https://github.com/tldr-pages/tldr-python-client), which is customizable and has lots of features and can be installed via `pip3`.
```
I think `which is supported by the tldr-pages project maintainers` is not nessecary, since that's already implied by `official`.

Reply:
Thats fine, however if one has any other good ideas I'll leave this open.

Reply:
Hrm. What about this?
```suggestion
You can also use the [official Python client](https://github.com/tldr-pages/tldr-python-client), which can be installed via `pip3`.
```

Reply:
I think it's good to know the difference between the python client and the others and `customizable and has lots of features` is objective enough to include it.

Reply:
How does one define `lots of features` though? Isn't the Node.js client customisable too with a config file?

Reply:
Hm, that wasn't as objective as I thought. How about 
```suggestion
You can also use the fully-featued [official Python client](https://github.com/tldr-pages/tldr-python-client), which can be installed via `pip3`.
```
to at least indicate that it doesn't lack any features?